File: app/src/main/java/test/platzi/com/raian/com/org/simpledynamit/model/country/Result.java

package test.platzi.com.raian.com.org.simpledynamit.model.country;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Result {

    @SerializedName("name")
    @Expose
    private String name;
    @SerializedName("code")
    @Expose
    private String code;
    @SerializedName("cities")
    @Expose
    private int cities;
    @SerializedName("locations")
    @Expose
    private int locations;
    @SerializedName("count")
    @Expose
    private int count;

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getCode() {
        return code;
    }

    public void setCode(String code) {
        this.code = code;
    }

    public int getCities() {
        return cities;
    }

    public void setCities(int cities) {
        this.cities = cities;
    }

    public int getLocations() {
        return locations;
    }

    public void setLocations(int locations) {
        this.locations = locations;
    }

    public int getCount() {
        return count;
    }

    public void setCount(int count) {
        this.count = count;
    }

}

File: app/src/main/java/test/platzi/com/raian/com/org/simpledynamit/model/country/Result.kt
package test.platzi.com.raian.com.org.simpledynamit.model.country

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class Result {

    constructor(name:String, code: String, cities: Int, locations: Int, count: Int)

    @SerializedName("name")
    @Expose
    var name: String? = null
    @SerializedName("code")
    @Expose
    var code: String? = null
    @SerializedName("cities")
    @Expose
    var cities: Int = 0
    @SerializedName("locations")
    @Expose
    var locations: Int = 0
    @SerializedName("count")
    @Expose
    var count: Int = 0

}
